File: src/main/java/com/github/rodbate/jetbrains/evaluate/refresher/services/MyProjectService.java
package com.github.rodbate.jetbrains.evaluate.refresher.services;

import com.github.rodbate.jetbrains.evaluate.refresher.MyBundle;
import com.intellij.openapi.project.Project;

public class MyProjectService {

    private final Project project;


    public MyProjectService(Project project) {
        this.project = project;
        System.out.println(MyBundle.message("projectService", project.getName()));
    }
}

File: src/main/kotlin/com/github/rodbate/intellijevaluaterefresher/services/MyProjectService.kt
package com.github.rodbate.intellijevaluaterefresher.services

import com.github.rodbate.intellijevaluaterefresher.MyBundle
import com.intellij.openapi.project.Project

class MyProjectService(project: Project) {

    init {
        println(MyBundle.message("projectService", project.name))
    }
}
